### DISCUSSION TITLE
Add the .env.example file

### DISCUSSION THREAD

**Discussion by andrii-bodnar** [directus/directus#14266] [Feature Requests]
Hi Directus team,

It would be nice if you can add a sample `.env.example` file with dummy data.

This should simplify the project initializing for local development.

Thanks!

**Comment by rijkvanzanten** [comment]
The example file was removed in https://github.com/directus/directus/pull/13352 in favor of relying on the generated .env from `npm init directus-project` including comments. https://docs.directus.io/self-hosted/config-options/ includes the reference of all options 👍🏻

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
